technical-embedded-ui/tpp-ui/src/app/app-routing.module.ts

```diff
@@ -16,11 +16,13 @@ const routes: Routes = [
   {path: 'parameters/provide-more/:executionId', component: ParametersInputComponent},
   {path: 'parameters/provide-psu-password/:executionId', component: ProvidePsuPasswordComponent},
   {path: 'parameters/select-sca-method/:executionId', component: SelectScaMethodComponent},
-  {path: 'parameters/report-sca-result/:executionId', component: ReportScaResultComponent}
+  {path: 'parameters/report-sca-result/:executionId', component: ReportScaResultComponent},
+  {path: '**', component: InitialRequestComponent}
```

Comment:
Routing all requests to the initial component is quite confusing. 
- In case of any error that is not handled by a router, you will see the initial page. 
- The `initial request` page was intentionally not defaulted one, as there should be a welcome page that shows you either swagger or technical-ui bundled in menu

Reply:
ok, so I only route / to InitialRequest

Reply:
I return to original state